test/e2e/e2e.sh

```diff
@@ -470,8 +473,37 @@ then
   exit 1
 fi
 
+# Webhook should reject invalid resources (v1beta1 version)
 if ! kubectl create --dry-run=server -f- <<'EOF' 2>&1 | grep -qF 'unknown time-slice interval'
 apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaim
+metadata:
+  name: webhook-test-v1beta1
+spec:
+  devices:
+    requests:
+    - name: ts-gpu
+      exactly:
+        deviceClassName: gpu.example.com
+    config:
+    - requests: ["ts-gpu"]
+      opaque:
+        driver: gpu.example.com
+        parameters:
+          apiVersion: gpu.resource.example.com/v1alpha1
+          kind: GpuConfig
+          sharing:
+            strategy: TimeSlicing
+            timeSlicingConfig:
+              interval: InvalidInterval
+EOF
+then
+  echo "Webhook did not reject v1beta1 ResourceClaim invalid GpuConfig with the expected message"
```

Comment:
Could we also mention the API version of the ResourceClaim in the message for the v1 test below like we do here?